structuredefinitions/Extension-England-OrganisationRole.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-OrganisationRole" />
+  <url value="https://fhir.nhs.uk/England/StructureDefinition/Extension-England-OrganisationRole" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandOrganisationRole" />
+  <title value="Extension England Organisation Role" />
+  <status value="draft" />
+  <experimental value="false" />
+  <date value="2023-10-16" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+      <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="An Extension to describe what an Organisation does and whether this is the primary role of the Organisation." />
+  <purpose value="An Extension to describe what an Organization does and whether this is the primary role of the Organisation." />
```

Comment:
Organization -> Organisation. This is for consistency with how it's spelt elsewhere